### PR TITLE
Adds some stunprod feedback

### LINKED ISSUE #20885: Stunprods don't work.
(WEB REPORT BY: subs455 REMOTE: 172.93.109.202:7777)
# Revision
98eed8c1e3d6ad26df1a424f12de40d802c6fd95

# Description
Stun Prods don't work. Assembling them and trying to activate them tells you it is out of charge, despite there being a 100% charged power cell in it.
# Steps to Reproduce
Build a stun pod and try to turn it on
# What you Expected
The stunprod to turn on and work normally.
# What Actually Happened
It stays off.

### PR DESCRIPTION
**Pull request by Zth--**
I'm bored, have you noticed already?

Closes #20885

:cl: 
- tweak: Stunprods now give a message when the cell is not powerful enough to turn it on: "[src] clicks but nothing happens. Something must be wrong with the battery."

### CODE REVIEW

File: code/game/objects/items/weapons/stunbaton.dm [hunk 1]
```diff
@@ -143,6 +143,8 @@
 		if(!bcell)
 			user.simple_message("<span class='warning'>[src] does not have a power source!</span>",
 				"<span class='warning'>[src] has no pulse and its soul has departed...</span>")
+		else if (bcell.maxcharge < hitcost)
+			user.simple_message("<span class='warning'>[src] clicks but nothing happens. You suspect something's wrong with the battery</span>")
```
Comment (SonixApache) [inline code/game/objects/items/weapons/stunbaton.dm]: you missed a period

File: code/game/objects/items/weapons/stunbaton.dm [hunk 2]
```diff
@@ -143,6 +143,8 @@
 		if(!bcell)
 			user.simple_message("<span class='warning'>[src] does not have a power source!</span>",
 				"<span class='warning'>[src] has no pulse and its soul has departed...</span>")
+		else if (bcell.maxcharge < hitcost)
+			user.simple_message("<span class='warning'>[src] clicks but nothing happens. You suspect something's wrong with the battery.</span>")
```
Comment (Kharkov72) [inline code/game/objects/items/weapons/stunbaton.dm]: I think "Something must be wrong with the battery" flows better than "You suspect something's wrong with the battery"

Comment (Kurfursten) [reply code/game/objects/items/weapons/stunbaton.dm]: use to_chat instead

**Comment by Kurfursten** [comment]
needs :cl:

**Comment by SonixApache** [comment]
does it really

**Comment by Kurfursten** [comment]
Non backend
Non administration

**Comment by Zth--** [comment]
... ... ...
ok

**Pull request merged**